pygame-blob/instructor_notes_roundtable.md

```diff
@@ -0,0 +1,23 @@
+*** DRAFT: Until further discussion about Roundtables** Used this (C14 roundatable)[https://github.com/gSchool/ada-developer-skills/blob/9044b8bccaefe1a5e43e6c83fb486a84b55f913b/02-unit2-roundtable-guides/01-wk09-rails-day-1.md] as a template.
+
+**Instructor Info**:
+
+The first part of the project will be an autograded, submission in Learn (students will just need to submit repl.it link)
+
+Fork from this (folder)[https://repl.it/repls/folder/PyBlobs] with links to solutions and starter kits.
+
+**Roundtable Agenda**:
+
+General:
+- Start class with with show-your-blobs from submission link
```

Comment:
My understanding is the showing off blobs will be in the round table the day after the round table with the refactor livecode as a brief intro. Students won't have made blobs with special effects until after the initial round table.